src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -101,6 +106,10 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
+  getImageThumbnail(thumbnail) {
+    return thumbnail ? (<img src={`${this.props.courseDetails.base_url}${thumbnail}`} alt="Description not available" />) : (<img src={undefined} alt="Preview not available" />);
```

Comment:
Nit (well, more like an vague opinion): it might be clearer to define `src` and `alt` outside of the markup and then return the `<img>` tag. e.g.:

```
if (thumbnail === undefined) {
   alt = ...
   src = ...
} else {
   ...
}
return (<img ... />);
```

Maybe too verbose though? It's up to you!